worker/src/RTC/DataConsumer.cpp

```diff
@@ -66,9 +67,12 @@ namespace RTC
 		// paused is set to false by default.
 		this->paused = data->paused();
 
-		for (const auto subchannel : *data->subchannels())
+		if (flatbuffers::IsFieldPresent(data, FBS::Transport::ConsumeDataRequest::VT_SUBCHANNELS))
```

Comment:
This check wasn't needed before. Why is it now?

Reply:
Because `subchannels` is an optional field, and not providing it (which is legitimate for an optional field) would crash. 

[Here](https://github.com/versatica/mediasoup/blob/flatbuffers/node/src/Transport.ts#L1730) you added an empty  array if none was provided, but not providing it, which is legitimate, would crash.

Reply:
True